src/pages/ServicesTests.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+
+import {
+  FetchResult,
+  WeatherResponseFormat,
+  ClimateDetails,
+} from "../services/services.inteface";
+
+import {
+  RequestByName,
```

Comment:
A common pattern is that function names should start with lower case letters. Components and interfaces can start with uppercase letters.